### PR TITLE
Minor fix in Console test

### PR DESCRIPTION
**Pull request by tarekgh**
This is a minor fix in the the console encoding test and added a small logging to the check to help telling about the used encoding in the test

### CODE REVIEW

**Comment by tarekgh** [comment]
CC @stephentoub @danmosemsft 

#14917

File: src/System.Console/tests/ConsoleEncoding.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public void TestEncoding(string inputString)
                 }
             }
 
-            Assert.Equal(inString, outString);
+            Assert.True(inString.Equals(outString), $"Encdoing: {Console.InputEncoding}, codepage: {Console.InputEncoding.CodePage}, expected: {inString}, Actual: {outString} ");
         }
```
Comment (danmosemsft) [inline src/System.Console/tests/ConsoleEncoding.cs]: Typo

File: src/System.Console/tests/ConsoleEncoding.cs [hunk 2]
```diff
@@ -68,7 +68,7 @@ public void TestEncoding(string inputString)
                 }
             }
 
-            Assert.Equal(inString, outString);
+            Assert.True(inString.Equals(outString), $"Encdoing: {Console.InputEncoding}, codepage: {Console.InputEncoding.CodePage}, expected: {inString}, Actual: {outString} ");
```
Comment (stephentoub) [inline src/System.Console/tests/ConsoleEncoding.cs]: Nit: Encdoing => Encoding
Nit: either capitalize codepage and expected or lowercase Actual, for consistency

**Comment by tarekgh** [comment]
thanks @stephentoub @danmosemsft for your review.

**Pull request merged**